docs/resources/cfw_black_white_list.md

```diff
@@ -0,0 +1,76 @@
+---
+subcategory: "Cloud Firewall (CFW)"
+---
+
+# huaweicloud_cfw_black_white_list
+
+Manages a CFW black white list resource within HuaweiCloud.
+
+## Example Usage
+
+```hcl
+variable "object_id" {}
```

Comment:
it's better to show how to get the object_id by [huaweicloud_cfw_firewalls](https://registry.terraform.io/providers/huaweicloud/huaweicloud/latest/docs/data-sources/cfw_firewalls)

Reply:
fixed